packages/jellyfish-api-core/src/category/governance.ts

```diff
@@ -31,6 +31,13 @@ export enum VoteDecision {
   NEUTRAL = 'neutral'
 }
 
+export enum VoteResult {
+  YES = 'YES',
+  NO = 'NO',
+  NEUTRAL = 'NEUTRAL',
+  Unknown = 'Unknown'
```

Comment:
Do we want to standardise with the other enums? Either `UPPERCASE`, `PascalCase` or `lowercase` ?

Reply:
Updated to use `UPPERCASE` for both key and values

Reply:
How about `Unknown`?

Reply:
on `jellyfish-api-core`, i am keeping `Unknown` to match what is returned from the `ain` side ([ref](https://github.com/DeFiCh/ain/blob/master/src/masternodes/proposals.cpp#L39-L48))
on `whale-api` i am mapping to return the uppercase `UNKNOWN`